celements-model/src/main/java/com/celements/model/util/ModelUtils.java

```diff
@@ -158,7 +158,7 @@
    * normalizeLanguage("  ")      = ""
    * normalizeLanguage("en")      = "en"
    * normalizeLanguage("DE_ch")   = "de_CH"
-   * normalizeLanguage("default") = "default"
+   * normalizeLanguage("default") = ""
```

Comment:
we appearently already expect in our code base to return return `""` for the input `"default"`, hence changing the behaviour in order to avoid breaking code.

Reply:
+1